core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1022,45 +1022,63 @@
   COMPACTOR_MINTHREADS_TIMEOUT("compactor.server.threads.timeout", "0s", PropertyType.TIMEDURATION,
       "The time after which incoming request threads terminate with no work available.  Zero (0) will keep the threads alive indefinitely."),
+  @Experimental
   COMPACTOR_THREADCHECK("compactor.server.threadcheck.time", "1s", PropertyType.TIMEDURATION,
       "The time between adjustments of the server thread pool."),
+  @Experimental
   COMPACTOR_MAX_MESSAGE_SIZE("compactor.server.message.size.max", "10M", PropertyType.BYTES,
       "The maximum size of a message that can be sent to a tablet server."),
   // CompactionCoordinator properties
+  @Experimental
   COORDINATOR_PREFIX("coordinator.", null, PropertyType.PREFIX,
       "Properties in this category affect the behavior of the accumulo compaction coordinator server."),
+  @Experimental
   COORDINATOR_THRIFTCLIENT_PORTSEARCH("coordinator.thrift.port.search", "false",
       PropertyType.BOOLEAN,
       "If the ports above are in use, search higher ports until one is available"),
+  @Experimental
   COORDINATOR_CLIENTPORT("coordinator.port.client", "9100", PropertyType.PORT,
       "The port used for handling Thrift client connections on the compaction coordinator server"),
+  @Experimental
   COORDINATOR_MINTHREADS("coordinator.server.threads.minimum", "1", PropertyType.COUNT,
       "The minimum number of threads to use to handle incoming requests."),
+  @Experimental
   COORDINATOR_MINTHREADS_TIMEOUT("coordinator.server.threads.timeout", "0s",
       PropertyType.TIMEDURATION,
       "The time after which incoming request threads terminate with no work available.  Zero (0) will keep the threads alive indefinitely."),
+  @Experimental
   COORDINATOR_THREADCHECK("coordinator.server.threadcheck.time", "1s", PropertyType.TIMEDURATION,
       "The time between adjustments of the server thread pool."),
+  @Experimental
   COORDINATOR_MAX_MESSAGE_SIZE("coordinator.server.message.size.max", "10M", PropertyType.BYTES,
       "The maximum size of a message that can be sent to a tablet server."),
+  @Experimental
   COORDINATOR_DEAD_COMPACTOR_CHECK_INTERVAL("coordinator.server.compactor.dead.check.interval",
       "5m", PropertyType.TIMEDURATION, "The interval at which to check for dead compactors."),
+  @Experimental
   COORDINATOR_FINALIZER_TSERVER_NOTIFIER_MAXTHREADS("coordinator.server.finalizer.threads.maximum",
       "5", PropertyType.COUNT,
       "The maximum number of threads to use for notifying tablet servers that an external compaction has completed."),
+  @Experimental
   COORDINATOR_FINALIZER_COMPLETION_CHECK_INTERVAL("coordinator.server.finalizer.check.interval",
       "60s", PropertyType.TIMEDURATION,
       "The interval at which to check for external compaction final state markers in the metadata table."),
+  @Experimental
   COORDINATOR_TSERVER_COMPACTION_CHECK_INTERVAL(
       "coordinator.server.tserver.compaction.check.interval", "1m", PropertyType.TIMEDURATION,
```

Comment:
Is `coordinator.server` redundant, since a coordinator *is* a server?

Reply:
Removed "server" from new property names in 42d0c8445d964f079f39ef4c50d05559191982a3